packages/material-ui/src/utils/focusVisible.test.js

```diff
@@ -2,7 +2,7 @@ import { assert } from 'chai';
 import React from 'react';
 import ReactDOM from 'react-dom';
 import { createMount } from '@material-ui/core/test-utils';
-import { teardown, useIsFocusVisible } from './focusVisible';
+import { teardown as teardownFocusVisible, useIsFocusVisible } from './focusVisible';
```

Comment:
`teardown` and `setup` are mocha globals for the BDD style. Since you can mix them the eslint plugin thinks you're calling a mocha global. I don't think it's a false positive but a unhelpful error message. What it should warn about is shadowing of mocha globals.